packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Unreleased
```

Comment:
If we can add a `# v0.1.4` here that'd be great

Reply:
done